src/features/home/components/modules/video/useVerticalVideoPlayer.ts

```diff
@@ -47,7 +47,8 @@ export const useVerticalVideoPlayer = ({
       setVideoState(state as PlayerState)
       setShowErrorView(false)
       if (state === PlayerState.ENDED) {
-        analytics.logHasSeenAllVideo(moduleId, currentVideoId)
+        const analyticsAllVideo = { moduleId, youtubeId: currentVideoId }
+        analytics.logHasSeenAllVideo({ ...analyticsAllVideo })
```

Comment:
pour quelles raisons auraient-on besoin de déconstruire et reconstruire un objet qui ne sera pas muté ?

```suggestion
        analytics.logHasSeenAllVideo(analyticsAllVideo)
```